### DISCUSSION TITLE
Mdmsv2

### DISCUSSION THREAD

**Discussion by radheshjoshi** [egovernments/Digit-Core#391] [DIGIT Platform]
I am getting error while creating data in mdmsv2 against the defined schema. The error i am getting is in mdmsdatavalidator as it is not validating Reference value provided.

**Comment by shashwat-egov** [comment]
Hi, thanks for reaching out to us. Can you provide more details regarding what schema you have created and what data you are trying to create? Also, with what I could infer, the error could be either due to misconfiguration in terms of "x-ref-schema" key in the child schema or you are not passing the unique identifier correctly in the child data which you are creating.

**Reply by radheshjoshi** [reply]
Hi, resolved the issue as i was not creating parent schema.
